data/metric/f1.json

```diff
@@ -0,0 +1 @@
+{"original file path": "/Users/daniel/.cache/huggingface/datasets/downloads/0adc5237d12211ec833346d633c0cf48092156ccae530a9ba4d121fef21a4e21.c7ed216fa7dfdaea390966614fb04835936577cfd0983a068e4d6bfe13cb8bcc.py", "local file path": "/Users/daniel/.cache/huggingface/modules/datasets_modules/metrics/f1/1ae7ede7c974ce472c931da73bdb2e3f0e1044a996d2348b328773d44dd1847b/f1.py"}
```

Comment:
what's this?:)

Reply:
TLDR: I can take it out

The way to understand what a metric in huggingface does is to generate it programmatically with: 

1. list_metrics() 
2. or if you know which one you need info then inspect_metrics('f1')

This generates the source code of that metric to a specified folder along with this json.

I have kept it in, in case we want to modify the f1 scoring function and want to understand what's in the background.
I have searched for the documentation of it on huggingface but its not there (probably could have looked at it on github?)